client/lib/titan/get-titan-product-name.js

```diff
@@ -1,18 +1,10 @@
 /**
  * External dependencies
  */
-import i18n from 'i18n-calypso';
-
-/**
- * Internal dependencies
- */
-import { isEnabled } from '@automattic/calypso-config';
+import { translate } from 'i18n-calypso';
 
 export function getTitanProductName() {
-	if ( isEnabled( 'titan/phase-2' ) ) {
-		return i18n.translate( 'Email', {
-			comment: 'Email is the name of a WordPress.com product, not just a capitalization of "email"',
-		} );
-	}
-	return i18n.translate( 'Titan Mail' );
+	return translate( 'Professional Email', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **31** times:
`translate( 'Professional email address' )` *ES Score: 7*